__tests__/topics.test.js

```diff
@@ -0,0 +1,98 @@
+const request = require('supertest');
+const db = require('../db/connection.js');
+const app = require('../app.js');
+
+describe('/api/topics', () => {
+  describe('we have to get all topics with slug and description', () => {
+    test('200: status code and contain the expected data type and fields', async () => {
+      const res = await request(app).get('/api/topics');
+      expect(res.statusCode).toBe(200);
+      expect(Array.isArray(res.body)).toBe(true);
+      res.body.forEach(topic => {
```

Comment:
don't forget to make sure that any arrays you're iterating over in a test aren't empty (ie. checking the length), otherwise an empty array will give you a false positive on this test